apps/studio/components/interfaces/Organization/BillingSettings/BillingBreakdown/UpcomingInvoice.tsx

```diff
@@ -195,33 +232,39 @@ const UpcomingInvoice = ({ slug }: UpcomingInvoiceProps) => {
                 </Collapsible>
               ))}
 
+            {computeCredits && (
+              <tbody>
+                <tr className="border-b">
+                  <td className="py-2 text-sm max-w-[200px]">
+                    <span className="mr-2">{computeCredits.description}</span>
+                    <InvoiceTooltip
+                      text="Paid plans come with $10 in Compute Credits to cover one Starter instance or parts of any other instance. Compute Credits are given to you not only for the first month but for every month while you are on a paid plan."
+                      linkRef="https://supabase.com/docs/guides/platform/org-based-billing#compute-credits"
+                    />
+                  </td>
+                  <td className="py-2 text-sm text-right" colSpan={3}>
+                    ${computeCredits.amount}
```

Comment:
Is `amount` properly formatted with thousand separators and currency?

Reply:
Good catch. Wasn't the case.
Using Javascript's `Intl.NumberFormat`now brings one change though. The minus sign for a negative amount is positioned before the dollar sign now instead of after the dollar sign. But that seems to be the standard way of doing it (and that's also how Stripe does it).
Once you say "OK" to this I'll use this helper function for other pages (e.g. usage breadown) as well.

Reply:
Makes sense, commented below regarding extracting the function and initializing